source/components/organisms/Step/Step.js

```diff
@@ -132,9 +133,35 @@ function Step({
         formNavigation.back();
       };
 
+  const [returnScrollY, setReturnScrollY] = useState(0);
+  const scrollRef = React.useRef();
+
+  const handleFocus = (e, isSelect = false) => {
```

Comment:
I'm not the most familiar with `useCallback`, but I'm assuming that since we'd be depending on `setReturnScrollY` and we're calling it in the hook, that we'd end up in an infinite loop?

I don't believe that `scrollRef` gets any different value on re-renders either.

Reply:
Calling `setReturnScrollY` shouldn't modify itself, so `useCallback` wouldn't need to be re-run. I think it works basically like a cache for the function itself, as otherwise the function gets re-constructed in memory on each re-render, which can be bad for performance.

Again I'm not 100% sure so we can skip it for now.